test/integration/sonobuoy_integration_test.go

```diff
@@ -48,16 +48,16 @@ func findSonobuoyCLI() (string, error) {
 
 // runSonobuoyCommandWithContext runs the Sonobuoy CLI with the given context and arguments.
 // It returns any encountered error and the stdout and stderr from the command execution.
-func runSonobuoyCommandWithContext(ctx context.Context, t *testing.T, args string) (bytes.Buffer, bytes.Buffer, error) {
-	var stdout, stderr bytes.Buffer
+func runSonobuoyCommandWithContext(ctx context.Context, t *testing.T, args string) (bytes.Buffer, error) {
+	var combinedOutput bytes.Buffer
```

Comment:
When testing anything we really care about what the users see; rarely exactly where it goes. Separating out the stdout/stderr was much harder to test 'what does the user see...'